File: src/wsb/merito/po/banking/TestBanking.java
package wsb.merito.po.banking;

public class TestBanking
{

    public static void main(String[] args) {


        Customer klient = null;
        Account account = null;

        klient = new Customer("Simms", "Jane");
        account = new Account(500);
        klient.setAccount(account);

        System.out.println(klient);

        klient = new Customer("Bryant", "Owen");
        account = new Account(200);
        klient.setAccount(account);

        System.out.println(klient);

    }


}

File: _Python/banking/test_banking.py
# Importujemy klasy z naszego pliku banking.py
from banking import Account, Customer

def test_bank_account():
    print("=== TestBankAccount ===")
    
    # Tworzenie obiektu (bez słowa kluczowego 'new')
    account = Account(100.50)
    print(account)
    
    account.withdraw(300)
    account.deposit(-50)
    print(account)
    
    account.withdraw(120)
    account.deposit(50)
    print(account)


def test_banking():
    print("\n=== TestBanking ===")
    
    # W Pythonie nie musimy deklarować zmiennych (np. Customer klient = null;) przed ich użyciem.
    
    klient = Customer("Simms", "Jane")
    account = Account(500)
    klient.set_account(account)
    print(klient)

    klient = Customer("Bryant", "Owen")
    account = Account(200)
    klient.set_account(account)
    print(klient)


# Odpowiednik public static void main(String[] args) w Javie
if __name__ == "__main__":
    test_bank_account()
    test_banking()